fedmsg.d/sse.py

```diff
@@ -9,6 +9,24 @@
     "fmn.sse.webserver.tcp_port": 8080,
     "fmn.sse.webserver.log": "sse.log",
 
+    # A regular expression using the standard Python re syntax that defines a
+    # whitelist of queues exposed by the SSE server.
+    'fmn.sse.webserver.queue_whitelist': '.+\.id\.fedoraproject\.org$',
+
+    # A regular expression using the standard Python re syntax that defines a
+    # blacklist for queues exposed by the SSE server. Any queue name that is
+    # matched by the regular expression will return a HTTP 403 to the client.
+    #
+    # Note: This is applied _after_ the whitelist so if the queue is matched
+    # by both regular expressions, the queue _will not_ be served.
+    'fmn.sse.webserver.queue_blacklist': None,
```

Comment:
I couldn't think of a particularly good reason to ever use a blacklist, but I thought that someone at some time might want to, and when that time comes, they probably don't want to wait on me to add it. It seemed super simple so I just did it in one pass.